inc/admin/dashboard/class-networkdashboard.php

```diff
@@ -55,4 +56,75 @@ protected function getTotalNumberOfBooks(): int {
 			)
 		);
 	}
+
+	protected function getNetworkChecklist() : array {
+		$items = [
+			[
+				'title' => __( 'Brand your network', 'pressbooks' ),
+				'link' => admin_url( 'customize.php' ),
+				'description' => __( 'Display your institutional logos and colors across your network', 'pressbooks' ),
+				'checked' => get_network_option( null, 'network_checklist_pressbooks_branding', false ),
+			],
+			[
+				'title' => __( 'Customize your homepage', 'pressbooks' ),
+				'link' => admin_url( 'edit.php?post_type=page' ),
+				'description' => __( 'Edit the textboxes and menu links on your homepage' ),
+				'checked' => get_network_option( null, 'network_checklist_customize_homepage', false ),
+			],
+			[
+				'title' => __( 'Review network settings', 'pressbooks' ),
+				'link' => network_admin_url( 'admin.php?page=pb_network_analytics_options' ),
+				'description' => __( 'Adjust defaults for new books and user permissions ' ),
+				'checked' => get_network_option( null, 'network_checklist_review_network', false ),
+			],
+			[
+				'title' => __( 'Configure Google Analytics', 'pressbooks' ),
+				'link' => network_admin_url( 'settings.php?page=pb_network_analytics_options#tabs-3' ),
+				'description' => __( 'Get more insight into your network’s web traffic' ),
+				'checked' => get_network_option( null, 'network_checklist_google_analytics', false ),
+			],
+		];
+
+		// Check if either SSO plugin is activated
```

Comment:
@arzola We can probably improve the logic here? DRY would be helpful, all that changes if CAS is active instead of SAML is the link URL ...